scripts/jei/mod/rats.zs

```diff
@@ -0,0 +1,54 @@
+#modloaded rats
+#priority -1
+//priority should be lower than scripts/mods/ratsprocessing.zs
+
+import crafttweaker.liquid.ILiquidStack;
+import crafttweaker.item.IItemStack;
+import crafttweaker.item.IIngredient;
+import crafttweaker.oredict.IOreDictEntry;
+import mods.requious.Assembly;
+import mods.requious.AssemblyRecipe;
+import mods.requious.SlotVisual;
+
+import scripts.mods.ratsprocessing;
+import scripts.jei.requious;
+
+val x = <assembly:rats_consuming> as Assembly;
+
+x.addJEICatalyst(<minecraft:spawn_egg>.withTag({EntityTag: {id: "rats:rat"}}));
```

Comment:
Use `<entity:>.asSoul()` and `<entity:>.asEgg()`. This using new cool features from `scripts/lib/expansions.zs`.
This will ensure that entity bracket captures are checked on compiling stage and also looks more clear than manual ID in NBT.